harmonica/tests/test_prism.py

```diff
@@ -52,6 +52,23 @@ def test_invalid_prisms():
         _check_prisms(np.atleast_2d([w, e, s, n, top, bottom]))
 
 
+@pytest.mark.use_numba
+def test_disable_checks():
+    "Check if the disable_checks flag works properly"
+    valid_prism = [0, 10, 0, 10, -10, 0]
+    invalid_prism = [0, 10, 0, 10, 0, -10]
+    density = 100
+    coordinates = [0, 0, 0]
```

Comment:
```suggestion
    coordinates = [0, 0, 0]
    # By default, an error should be raised for invalid input
    with pytest.raises(ValueError):
        prism_gravity(coordinates, invalid_prism, density, field="potential")
```

Added a test that the error is still raised by default. Just to complement the existing tests.